### DISCUSSION TITLE
maze_world.launch.py 실행시 rviz상으로 gcamp_world가 출력됩니다.

### DISCUSSION THREAD

**Discussion by ckj0123** [Road-Balance/gcamp_ros2_basic#11] [Q&A] [answered]
ubuntu20.04에서 ros2 foxy를 사용중입니다.

![Screenshot from 2022-02-16 17-47-28](https://user-images.githubusercontent.com/95273933/154228528-16112968-bad6-4f3f-86a9-b297284b11bd.png)
우선 다음과 같이 maze_world.launch.py를 실행시켰습니다.

![Screenshot from 2022-02-16 17-47-53](https://user-images.githubusercontent.com/95273933/154228605-1e3a1934-805b-4306-bbc9-84d1b1628059.png)
첫째, gazebo상으론 maze가 생성됬지만, 현재 robot이 gazebo상에 생성되지 않습니다.

![Screenshot from 2022-02-16 17-48-40](https://user-images.githubusercontent.com/95273933/154228687-f7918f2c-c1a2-4c3d-89da-4d48c30deeba.png)
둘째, rviz상으로 gcamp_world의 벽부분이 image를 통해 생성된 것을 확인할 수 있습니다.
rviz에서 사용한 topic list
laser_scan topic = diffbot/scan
image                 = /diffbot/camera_sensor/image_raw
나머지 topic들은 변경하지 않았습니다. (tf 체크 표시만 check로 바꿨습니다.)

![Screenshot from 2022-02-16 17-52-33](https://user-images.githubusercontent.com/95273933/154229368-0f7e7a62-1b79-4e38-ab49-ffc1a93b7902.png)
파이썬 파일에서도 world_file_name의 이름과 아래의 world_file경로는 제대로 생성되어있다 생각됩니다.

maze_world가 rviz상으로 제대로 출력되려면 어떻게 해야하는지 여쭙습니다.

감사합니다.

**Comment by kimsooyoung** [comment, marked as answer]
문제없어 보입니다. 다시 확인 부탁드립니다.

![image](https://user-images.githubusercontent.com/12381733/154946253-549e192b-265d-4da8-a083-f9ab974e9d2e.png)
